packages/discovery-provider/src/tasks/entity_manager/entities/social_features.py

```diff
@@ -99,6 +101,33 @@ def create_social_record(params: ManageEntityParameters):
             challenge_event, params.block_number, params.block_datetime, params.user_id
         )
 
+    # dispatch verified cosign challenge
+    if (
+        params.action in [Action.SAVE, Action.REPOST]
+        and params.entity_type == EntityType.TRACK
+        and params.existing_records[EntityType.TRACK][params.entity_id].remix_of
+    ):
+        remixed_track = (
+            params.session.query(Track)
+            .join(Remix, Remix.parent_track_id == Track.track_id)
+            .filter(Remix.child_track_id == params.entity_id)
+            .first()
+        )
+        remixer = params.existing_records[EntityType.TRACK][params.entity_id].owner_id
+        cosigner = params.existing_records[EntityType.USER][params.user_id]
+        if (
+            remixed_track
+            and remixed_track.owner_id == params.user_id
+            and cosigner.is_verified
```

Comment:
trying to think if there's a way someone could abuse this since the `is_verified` constraint isn't enforced in the challenge updater itself. i think we're good tho? there's no way to get here from an cosign transaction, would have to abuse redis.

Reply:
yeaaa i think it's ok like this